libbz2-rs-sys/src/bzlib.rs

```diff
@@ -1451,7 +1450,7 @@ fn un_rle_obuf_to_output_fast(strm: &mut BzStream<DState>, s: &mut DState) -> bo
                 BZ_GET_FAST_C!(c_k0);
                 c_nblock_used += 1;
 
-                break;
+                continue 'return_notr;
```

Comment:
Why was this changed?

Reply:
to make it more consistent with the other branches. The control flow here is not a special case, but analogous to the earlier cases.